packages/grid/x-data-grid/src/components/menu/columnMenu/simple/GridColumnMenuSimple.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Divider from '@mui/material/Divider';
+import { GridColumnMenuTypes } from '../../../../hooks/features/columnMenu';
+import { GridColumnMenuSimpleContainer } from './GridColumnMenuSimpleContainer';
+import { GridColumnMenuProps } from '../GridColumnMenuProps';
+import { GridColumnsMenuSimpleItem } from './GridColumnsMenuSimpleItem';
+import { GridFilterMenuSimpleItem } from './GridFilterMenuSimpleItem';
+import { HideGridColMenuSimpleItem } from './HideGridColMenuSimpleItem';
+import { SortGridMenuSimpleItems } from './SortGridMenuSimpleItems';
+import { GridColumnMenu } from '../GridColumnMenu';
+
+interface Props
+  extends Pick<
+    GridColumnMenuProps,
+    'hideMenu' | 'currentColumn' | 'open' | 'getVisibleColumnMenuItems'
+  > {}
+
+const defaultVisibleItems: Array<GridColumnMenuTypes['key']> = [
+  'sorting',
+  'filter',
+  'hideColumn',
+  'manageColumns',
+];
+
+const GridColumnMenuSimple = React.forwardRef<HTMLUListElement, Props>(
+  function GridColumnMenuSimple(props: Props, ref) {
+    const defaultMenuItems = {
+      sorting: <SortGridMenuSimpleItems />,
+      filter: <GridFilterMenuSimpleItem />,
+      hideColumn: <HideGridColMenuSimpleItem />,
+      manageColumns: <GridColumnsMenuSimpleItem />,
+      divider: <Divider />,
+    };
+
+    return (
+      <GridColumnMenuSimpleContainer ref={ref} {...props}>
+        <GridColumnMenu
+          // @ts-ignore as it's not getting proper types due to overrides from Pro/Premium
```

Comment:
To avoid it we can do the following:
```diff
diff --git a/packages/grid/x-data-grid/src/hooks/features/columnMenu/columnMenuInterfaces.ts b/packages/grid/x-data-grid/src/hooks/features/columnMenu/columnMenuInterfaces.ts
index 2cfc6d1..c90c795 100644
--- a/packages/grid/x-data-grid/src/hooks/features/columnMenu/columnMenuInterfaces.ts
+++ b/packages/grid/x-data-grid/src/hooks/features/columnMenu/columnMenuInterfaces.ts
@@ -23,6 +23,6 @@ export interface GridColumnMenuTypes {
 }
 
 export interface GridColumnMenuValue {
-  items: { [key in GridColumnMenuTypes['key']]: React.ReactNode };
+  items: { [key in GridColumnMenuTypes['key']]?: React.ReactNode };
   visibleItemKeys: Array<GridColumnMenuTypes['key']>;
 }

```

It's not perfect, but probably better than `@ts-ignore`